CONTRIBUTING

```diff
@@ -121,3 +121,287 @@ Bad Contributions
       something that kickstart needs to capture and provide to programs using
       it.
 
+
+KICKSTART DEPRECATION
+
+Over time commands and arguments change. New ones are added and old ones are
+eventually removed.  Kickstart's use of the term 'deprecation' can be a bit
+confusing.  At the time of this writing, version 3.31, a deprecated command
```

Comment:
I would remove ``Kickstart's use of the term 'deprecation' can be a bit confusing.`` it's mentioned in the next paragraph too.